manifests/globals.pp

```diff
@@ -83,7 +83,7 @@
             $fpm_service_name     = "php${globals_php_version}-fpm"
             $ext_tool_enable      = "/usr/sbin/phpenmod -v ${globals_php_version}"
             $ext_tool_query       = "/usr/sbin/phpquery -v ${globals_php_version}"
-            $package_prefix       = 'php7.0-'
+            $package_prefix       = "php${globals_php_version}-"
```

Comment:
This will break for php 5, where $default_php_version is "5.x" and no just "5" :/